bindgen/templates/CallbackInterfaceRuntime.cs

```diff
@@ -1,92 +0,0 @@
-{#/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */#}
-
-static class UniffiCallbackResponseCode {
-    public static int SUCCESS = 0;
-    public static int ERROR = 1;
-    public static int UNEXPECTED_ERROR = 2;
-}
```

Comment:
Since this was here before, perhaps we can move it out and reuse it in the new callback handling code?